src/docfx/build/moniker/MonikerProvider.cs

```diff
@@ -49,7 +49,7 @@ public int GetMonikerOrder(string moniker)
             {
                 return value;
             }
-            return int.MinValue;
+            return 0;
```

Comment:
Since `order` in optional, 0 is default as lowest order.
https://github.com/dotnet/docfx/blob/5b882493aa776dc37106cb6466b6178a81a7196c/src/docfx/lib/moniker/Moniker.cs#L17